CHANGELOG.md

```diff
@@ -2,9 +2,15 @@
 
 ## Unreleased
 
+### 🛑 Breaking changes 🛑
+
+- (Splunk) On Windows the `SPLUNK_*` environment variables were moved from the machine scope to the collector service scope.
+  It is possible that some instrumentations are relying on the machine-wide environment variables set by the installation. ([#3930](https://github.com/signalfx/splunk-otel-collector/pull/3930))
```

Comment:
nit, the aim of this statement could be misinterpreted as an ~ambiguous call to action. Maybe something like this throughout the other instances?

```suggestion
- (Splunk) On Windows the `SPLUNK_*` environment variables were moved from the machine scope to the collector service scope. This avoids collisions with other agents and instrumentation. ([#3930](https://github.com/signalfx/splunk-otel-collector/pull/3930))
```